### DISCUSSION TITLE
[deps] why locks neodim version

### DISCUSSION THREAD

**Discussion by ClSlaid** [ayamir/nvimdots#1138] [Q&A]
I notice that the version of neodim is lock, any reason to do that?

As I know, this ref of neodim.nvim is not compatible with Neovim version above https://github.com/neovim/neovim/commit/c0cb1e8e9437b738c8d3232ec4594113d2221bb2.

**Comment by ayamir** [comment]
https://github.com/ayamir/nvimdots/pull/1003

**Comment by charliie-dev** [comment]
#1142
